src/utils/constants.js

```diff
@@ -15,6 +15,8 @@ export const validationConfig = {
   inputErrorClass: "modal__input_type_error",
   errorClass: "modal__error_visible",
 };
+export const avatarEditModal = document.querySelector("#avatar-modal");
+export const avatarEditForm = avatarEditModal.querySelector("#avatar-form");
 
 // Create instances of FormValidator for each form
```

Comment:
Please, delete `this.toggleButtonState();` from line 69 in `FormValidator.js`

You need to **disable** the button only in block `then` after a **successful response** to be able to click it again if there is a server error.